books/spiders/books_info.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any
+import re
+
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class BooksInfoSpider(scrapy.Spider):
+    name = "books_info"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def __init__(self, **kwargs: Any):
+        super().__init__(**kwargs)
+        options = webdriver.ChromeOptions()
+        options.add_argument("--headless")
+        self.driver = webdriver.Chrome(options=options)
+
+    def close(self, reason) -> None:
+        self.driver.close()
+
+    def parse(self, response: Response, **kwargs):
+        for book in response.css(".product_pod"):
+            yield self.parse_detail_page(response, book)
```

Comment:
The `parse_detail_page` method is called with `response` and `book`, but it returns a dictionary. Ensure that this method is yielding the dictionary as an item, rather than returning it, to properly integrate with Scrapy's item pipeline.